devices.js

```diff
@@ -9922,6 +9922,55 @@ const devices = [
             await bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
         },
     },
+    {
+        zigbeeModel: ['eTRV0100'],
+        model: 'eTRV0100',
+        vendor: 'Danfoss',
+        description: 'Danfoss Ally Thermostat',
+        supports: 'temperature, heating system control',
+        fromZigbee: [
+           fz.eurotronic_thermostat,
```

Comment:
You probably don't want this one, it does some ugly shuffling of system_mode and parsing of custom eurotronic bits.

Reply:
I'll review the setup on the weekend and verify it works, then update the PR. I don't want to make any changes without checking the hardware.